src/eve-server/system/Container.cpp

```diff
@@ -298,35 +306,14 @@ void ContainerSE::Process() {
     }
 }
 
-void ContainerSE::Activate(int32 effectID)
-{
-    // check effectID, check current state, check current timer, set new state, update timer
-
-    /** @todo somehow notify client with one of these effects:
-     *  effectAnchorDrop = 649
-     *  effectAnchorLift = 650
-     *  effectAnchorDropForStructures = 1022
-     *  effectAnchorLiftForStructures = 1023
-     *
-     ** @todo  many more effects to send for.....look into later.
-     * effectOnlineForStructures = 901
-     *
-     ** @note  also note there are timers involved here...
-     */
-}
-
-void ContainerSE::Deactivate(int32 effectID)
-{
-    // check effectID, check current state, check current timer, set new state, update timer
-}
-
 void ContainerSE::AnchorContainer()
 {
     m_deleteTimer.Disable();
     m_contRef->SetAnchor(true);
 }
 
-void ContainerSE::EncodeDestiny( Buffer& into )
+//might whant to remove this
```

Comment:
This whole block can be removed since its handled by Structure. Ideally no derrived classes should need to override EncodeDestiny for structures.

Reply:
Right will do